crates/biome_css_formatter/tests/specs/scss/expression/map-pair-leading-comments.scss

```
@@ -0,0 +1,9 @@
+$foo:(property1:(),// comment 1
+// comment 2
+
+property2:1,/** comment 1 */
+/* comment 2 */
+property3:2);
+
+$line:(// keep
+key:(inner:value,),);
```

Stylelint (17.12.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)